scenarios/httpclient.benchmarks.yml

```diff
@@ -0,0 +1,44 @@
+﻿# Local run:
+# crank --config "C:\Users\knatalia\dev\git\Benchmarks\scenarios\httpclient.benchmarks.yml" --scenario httpclient-kestrel-get --profile local
+
+imports:
+  - https://github.com/aspnet/Benchmarks/blob/main/scenarios/aspnet.profiles.yml?raw=true
+
+variables:
+    serverPort: 5000
+    httpVersion: "1.1"
+    numberOfClients: 1
+    concurrencyPerClient: 12
+    http11MaxConnectionsPerServer: 1
+    http20EnableMultipleConnections: false
+    warmup: 15
+    duration: 30
+
+scenarios:
+  httpclient-kestrel-get:
+    application:
+      job: kestrel
+    load:
```

Comment:
It's a bit confusing to me that the "application" is kestrel and the "load" is httpClient.

The thing we want to measure here in httpClient. So why isn't the application httpClient?

What will happen if I turn on perf collection, for example? Will it collect a perf trace of kestrel or httpClient?

Reply:
When you enable collection or counters you explicitly set which "job" does it, and you can set both if you want.
Something like `--load.collect true` will work.

Reply:
Ok, then what's the difference between "application" and "load"? Are they just arbitrary names? If so can we call them something clearer?

Reply:
They are arbitrary, but they are mapped in the profiles to specific machines. I have an issue/task to create another indirection between names and target machines, which would allow use to change the names without impacting the profiles (aspnet-perf-win, aspnet-citrine-arm, ...)

Reply:
Interesting. What sort of hardware will the httpClient job end up running on, then? I assume we want this to run on the "big" hardware, meaning aspnet-perf-win etc... though it's also interesting to run it on smaller boxes, like 4 CPUs or 8 CPUs.

Reply:
You set it on each run. This is not the issue with this PR, when a crank command is run we specify the profile (target machines) in the crank arguments. And our continuous runs will do that on several machines, including Azure VMs.

Reply:
Okay, but I assume I want to use the profile "aspnet-perf-lin" or "aspnet-perf-win" typically, right? What will happen when I do that? Where will each job run and on what sort of hardware?

Reply:
@geoffkizer You are right, we need to invert the name until I am doing the mappings feature, because otherwise stating asp-perf-win will still run the client on a Linux box.

Reply:
I can rename them to whatever. I would just need to copy and edit the profiles from `aspnet.profiles.yml` instead of importing them. (until the mapping is implemented as @sebastienros mentioned)
Thanks @sebastienros  for pointing out that in `asp-perf-win` profile only `application: endpoints: - http://asp-perf-win:5001` is windows and `load: endpoints: - http://asp-perf-db:5001` is linux actually; I didn't look inside profiles closely enough so I wasn't aware. I will invert the machines. 

Interestingly, that might be connected to why HTTP/3 wasn't working on `asp-perf-win`

Also, @sebastienros what defines an order in which jobs are created? Is it the order here in yml in `scenarios` definition, or something else?

Reply:
If I haven't answered somewhere else, it's the order in the scenario. And `waitForExit` is blocking the next one from starting until it has finished. That's why a web app doesn't have `waitForExit: true`, once it's ready the next service starts.